src/builders/solr-query-builder.php

```diff
@@ -9,10 +9,21 @@
  */
 class SolrQueryBuilder implements QueryBuilder {
   /**
-   * Build a query string from provided search terms and search options
+   * Build a query string from provided search terms and search options.
+   * Returns a query string for a SOLR server.
+   *
+   * @param SearchTerm[]|null $searchTerms Search terms to search
+   * @param SearchApi\Models\SearchOptions|null $options Query options
    */
-  function build( $keywords, $options = null ) {
-    // TODO: This is just a placeholder
-    return '?q=collector%3Aurbanization&wt=json&indent=true';
+  function build( $searchTerms, $options = null ) {
+    $keywordStrings = '';
+
+    error_log( var_dump( $options ) );
```

Comment:
You could do some a simple type check, since we'll want $options to be an array you could:
```
// todo: actually do something with options in the future
$options = (array) $options;
```
or alternatively just disable that rule for this function with an annotation.